protocol-units/bridge/integration-tests/tests/client_l1move_l2move.rs

```diff
@@ -0,0 +1,455 @@
+
+	test_result
+}
+#[tokio::test]
+async fn test_eth_client_should_build_and_fetch_accounts() {
+	let config = Config::default();
+	let (eth_client_harness, _config, _anvil) = TestHarness::new_only_eth(config).await;
+
+	let expected_accounts = [
+		address!("f39fd6e51aad88f6f4ce6ab8827279cfffb92266"),
+		address!("70997970c51812dc3a010c7d01b50e0d17dc79c8"),
+		address!("3c44cdddb6a900fa2b585dd299e03d12fa4293bc"),
+		address!("90f79bf6eb2c4f870365e785982e1f101e93b906"),
+		address!("15d34aaf54267db7d7c367839aaf71a00a2c6a65"),
+		address!("9965507d1a55bcc2695c58ba16fb37d819b0a4dc"),
+		address!("976ea74026e726554db657fa54763abd0c3a0aa9"),
+		address!("14dc79964da2c08b23698b3d3cc7ca32193d9955"),
+		address!("23618e81e3f5cdf7f54c3d65f7fbc0abf5b21e8f"),
+		address!("a0ee7a142d267c1f36714e4a8f75612f20a79720"),
+	];
+
+	let provider = eth_client_harness.rpc_provider().await;
+	let accounts = provider.get_accounts().await.expect("Failed to get accounts");
+	assert_eq!(accounts.len(), expected_accounts.len());
+
+	for (account, expected) in accounts.iter().zip(expected_accounts.iter()) {
+		assert_eq!(account, expected);
+	}
+}
+
+#[tokio::test]
+async fn test_eth_client_should_deploy_initiator_contract() {
+	let config = Config::default();
+	let (_eth_client_harness, config, _anvil) = TestHarness::new_only_eth(config).await;
+
+	assert!(config.eth.eth_initiator_contract != "Oxeee");
+	assert_eq!(
+		config.eth.eth_initiator_contract, "0x8464135c8F25Da09e49BC8782676a84730C318bC",
+		"Wrong initiator contract address."
+	);
+}
+
+#[tokio::test]
+async fn test_eth_client_should_successfully_call_initialize() {
+	let config = Config::default();
+	let (_eth_client_harness, config, _anvil) = TestHarness::new_only_eth(config).await;
+	assert!(config.eth.eth_counterparty_contract != "0xccc");
+	assert_eq!(
+		config.eth.eth_counterparty_contract, "0x71C95911E9a5D330f4D621842EC243EE1343292e",
+		"Wrong initiator contract address."
+	);
+	assert!(config.eth.eth_weth_contract != "0xe3e3");
+	assert_eq!(
+		config.eth.eth_weth_contract, "0x948B3c65b89DF0B4894ABE91E6D02FE579834F8F",
+		"Wrong initiator contract address."
+	);
+}
+
+#[tokio::test]
+async fn test_eth_client_should_successfully_call_initiate_transfer_only_eth() {
+	let config = Config::default();
+	let (mut eth_client_harness, _config, _anvil) = TestHarness::new_only_eth(config).await;
+
+	let signer_address: alloy::primitives::Address = eth_client_harness.signer_address();
+
+	let recipient = HarnessMvtClient::gen_aptos_account();
+	let hash_lock: [u8; 32] = keccak256("secret".to_string().as_bytes()).into();
+	eth_client_harness
+		.eth_client
+		.initiate_bridge_transfer(
+			BridgeAddress(EthAddress(signer_address)),
+			BridgeAddress(recipient),
+			HashLock(EthHash(hash_lock).0),
+			Amount(AssetType::EthAndWeth((1, 0))), // Eth
+		)
+		.await
+		.expect("Failed to initiate bridge transfer");
+}
+
+#[tokio::test]
+async fn test_eth_client_should_successfully_call_initiate_transfer_only_weth() {
```

Comment:
These should be testing the MOVE.sol contracts seems like they are initialised currently with the old Initiator contracts. In other words this it `AtomicBridgeInitiator` -> COIN. Right?

Reply:
I didn't touch the Eth client tests, I think I had gotten it in my mind that we were using the E2E tests to tests the `.sol` contracts; I was just using the client tests to check the Movement client against the framework.

One thing I notice on `main` is that there aren't Eth tests in ` `client_movement_eth` anymore. Not sure why or how they disappeared.

Reply:
Don't worry about this, I'm going to make sure the .sol MOVE contrats are supported by the relayer in a seperate PR. Right now it only uses the old ABIs for WETH contracts.